includes/object-cache.php

```diff
@@ -14,7 +14,9 @@
 
 defined( '\\ABSPATH' ) || exit;
 
-if ( ! defined( 'WP_REDIS_DISABLED' ) || ! WP_REDIS_DISABLED ) :
+if ( ! defined( 'WP_REDIS_DISABLED' ) || ! WP_REDIS_DISABLED ) {
+    return;
+}
```

Comment:
This doesn't work, try disabling the object cache with it.

Reply:
🤦 Well it would work if I had actually done the necessary boolean conversion to the negative form. Tested that and it worked. Will push the new one in a second.

Reply:
Are you 100% sure that the `wp_cache_*()` functions are not being declared this way anyhow? I recall reverting this change once before.

Reply:
Functions will be defined before the code itself is run so returning early will still define the cache functions. Reverted the commit.